pages/common/xmllint.md

```diff
@@ -18,7 +18,7 @@
 
 `xmllint --format {{source_file.xml}}`
 
-- Check that a XML file meets the requirements of its DOCTYPE declaration:
+- Check that an XML file meets the requirements of its DOCTYPE declaration:
```

Comment:
This is wrong as well IMO however this can be desputed

Reply:
Most people read `XML` as `ex em el`, which then would require `an`. Same for `HTML`.